NatureArea.py

```diff
@@ -0,0 +1,245 @@
+from WikidataItem import WikidataItem
+import importer_utils as utils
+
+
+class NatureArea(WikidataItem):
+    """
+    Extension of WikidataItem to create nature area objects.
+
+    Handles both reserves and national parks.
+    """
+
+    def generate_ref_url(self):
+        url = "http://nvpub.vic-metria.nu/naturvardsregistret/rest/omrade/{}/G%C3%A4llande"
+        return url.format(self.raw_data["NVRID"])
+
+    def create_sources(self):
+        """Create the references for all statements."""
+        self.sources = {}
+        item_nr = self.items["source_nr"]
+        item_np = self.items["source_np"]
+        url = self.generate_ref_url()
+        self.sources["np"] = self.make_stated_in_ref(item_np,
+                                                     "2015-12-18",
+                                                     url)
+        self.sources["nr"] = self.make_stated_in_ref(item_nr,
+                                                     "2015-12-18",
+                                                     url)
+
+    def set_labels(self):
+        """
+        Add labels, optionally in multiple languages.
+
+        A number of items have the word natur/domän etc. reservat
+        included in the name. In this case, the name is
+        added only in Swedish. Otherwise, if it's a pure
+        geographical name, it is added in a number of Latin script
+        languages.
+        """
+        swedish_name = self.raw_data["NAMN"]
+        if ("reservat" in swedish_name.lower() or
```

Comment:
I've looked into it and there's a couple of variations, but they're all based on the word "naturreservat", so they're covered by this. Mostly "domänreservat", some "fågelreservat" and so on. A couple of the type "Lindreservatet" as well.

Done on the latter.